Add q3 roadmap (#6)
meilisearch/core-team: roadmaps/2021.md

## Q2

- [X] Start some benchmarks
- [X] Version the core-team repositories
- _Reported, waiting for internal decision_ Publish the core-team repositories to crates.io
- [X] Release the first RC of MeiliSearch v0.21.0 that integrates the new search engine (Transplant + Milli)
- _Reported to Q3_ Release v0.21.0 of MeiliSearch

## Q3

### Internal

- [ ] Release v0.21.0
- [ ] Publish core-team repositories (depending on internal decisions)
  - [ ] tokenizer
  - [ ] milli
  - [ ] lmdb-rs
- [ ] Cron job CI to run some SDKs tests
- [ ] MeiliSearch (e.g. Transplant) documentation
- [ ] Indexation benchmarks

### New features

- [ ] SortBy
- [ ] Geo search
- [ ] NDJSON

> - [ ] CSV

Author: ```suggestion
- [ ] CSV
- [ ] Improve analytics
```

## Q2

- [X] Start some benchmarks
- [X] Version the core-team repositories
- _Reported, waiting for internal decision_ Publish the core-team repositories to crates.io
- [X] Release the first RC of MeiliSearch v0.21.0 that integrates the new search engine (Transplant + Milli)
- _Reported to Q3_ Release v0.21.0 of MeiliSearch

## Q3

> 

Reviewer: ```suggestion
### Goals

- Reduce biggest user's deal-breakers.
- Start to stabilize the API format block by block until v1.0. 
- Understand our user's behaviors.
- Patch v0.21 indexing speed.
```

### Internal

- [ ] Release v0.21.0
- [ ] Publish core-team repositories (depending on internal decisions)
  - [ ] tokenizer
  - [ ] milli
  - [ ] lmdb-rs
- [ ] Cron job CI to run some SDKs tests
- [ ] MeiliSearch (e.g. Transplant) documentation

> - [ ] Indexation benchmarks
> 
> ### New features
> 
> - [ ] SortBy
> - [ ] Geo search
> - [ ] NDJSON
> - [ ] CSV
> - [ ] Improve analytics
> - [ ] Clean API
>   - [ ] Search
>   - [ ] Documents
>   - [ ] Updates
>   - [ ] Indexes

Reviewer: ```suggestion
- [ ] Indexation benchmarks (Patch v0.21 indexing speed)

### New features

#### Reduce biggest user's deal-breakers
- [ ] SortBy
- [ ] Geo search

#### Patch v0.21 indexing speed
- [ ] Indexing NDJSON file format
- [ ] Indexing CSV file format
- [ ] Indexer optimization

#### Understand our user's behaviors.
- [ ] Branch analytics at feature level (On those planned in this quarter)

#### Stabilize the API format block by block until v1.0.
- [ ] Clean API Format
  - [ ] Search
  - [ ] Documents
  - [ ] Updates
  - [ ] Indexes
```

> 
> ### Optimization
> 
> - [ ] Indexer

Reviewer: ```suggestion
```

Reviewer: I moved the optimization line under the objective of improving indexing performance! Thanks, your work @curquiza!
